help.md

```diff
@@ -0,0 +1,901 @@
+# Table of Contents
+<div class="toc">
+
+- About  
+- Layers  
+  - Creating New Layers
+  - Saving and Loading Layers
+  - Creating Layers from Other Layers
+- Layer Controls
+  - Layer Information
+  - Sorting
+  - Filtering
+    - Platform Filter
+  - Color Setup
+    - Tactic Row Background
+    - Scoring Gradient
+  - Hiding Disabled Techniques
+  - Showing or Hiding Sub-techniques
+  - Configuring the Layout
+    - Side Layout
+    - Flat Layout
+    - Mini Layout
+    - Showing IDs and Names
+    - Aggregate Scores
+      - Showing or Hiding Aggregate Scores
+      - Counting Unscored Techniques
+      - Choosing an Aggregate Function
+  - Legend
+- Technique Controls
+  - Disabling Techniques
+  - Assigning Manual Colors
+  - Scoring Techniques
+  - Adding Comments to Techniques
+  - Clearing Annotations on Techniques
+- Selecting Techniques
+  - Selecting with the Mouse
+  - Selection Behavior
+  - Search & Multiselect Interface
+- Customizing the Navigator
+  - Default Layers
+  - Disabling Features
+- Rendering Layers as SVG
+  - Measurement Units
+  - Configuring Image Size
+  - Configuring Text
+  - Customizing the Legend
+  - Display Settings
+- Exporting Layers to MS Excel
+- Notice
+
+</div>
+
+<span class="about"></span>
+# MITRE ATT&CK® Navigator Version 4.4
+<p>
+    The ATT&CK Navigator is designed to provide basic navigation and
+    annotation of ATT&CK matrices, something that people are already doing
+    today in tools like Excel. We've designed it to be simple and generic -
+    you can use the Navigator to visualize your defensive coverage, your
+    red/blue team planning, the frequency of detected techniques or anything
+    else you want to do. The Navigator doesn't care - it just allows you to
+    manipulate the cells in the matrix (color coding, adding a comment,
+    assigning a numerical value, etc.). We thought having a simple tool that
+    everyone could use to visualize the matrix would help make it easy to
+    use ATT&CK.
+</p><br>
```

Comment:
I realize it's a lot more work, but the inclusion of HTML tags for paragraphs is sort of un-markdown-y. HTML is valid markdown so it does work, but it's unnecessary to include and increases maintenance burden. Ideally these tags (`<p>` and `<br>`) should be removed altogether since an empty line between paragraphs is all that is actually required for paragraphs to render in Markdown.